### PR TITLE
Theme modifications

### PR DESCRIPTION
**Pull request by TheMorc**
I rewrote my original pixelperfect commit to be a bit less hacky and also fixed the System7 theme a bit more.

_I apologize for my IDE messing the files a bit more than it should be_

### CODE REVIEW

**Comment by morgant** [comment]
This looks much better! Many thanks!

On the System 7 theme, I think the close & resize buttons need to be nudged outward a bit more. Also, it seems that the horizontal lines don't need to have a lighter line beneath each (again, on System 7 theme), just the background color.

I too would like the menu bar height reduced to 20px, but it looks wrong with the default 14px font size that is selected. Changing `DEFAULTFS` & `DEFAULTFONT` in `mlvwm.h` to 12px font patterns looks better to me. Thoughts?

**Comment by TheMorc** [comment]
I'll try to do something about those stripes there.
And about those fonts, when i was messing with them i wasn't able to change font sizes but it'll maybe start to work.

Also there still are a few places that i would want to change.

**Comment by TheMorc** [comment]
So i tried to fix those stripes and i found out that the titlebar is taller on MacOS 8/9 than on System 7.
+ that font size you suggested.

File: mlvwm/borders.c [hunk 1]
```diff
@@ -488,30 +488,19 @@ void SetTitleBar( MlvwmWindow *t, Bool on_off )
...
-					DrawShadowBox( 4, lp-1, t->frame_w-11, 2, t->title_w, 1,
-								  Scr.Gray1GC, Scr.WhiteGC, SHADOW_ALL );
+                else
+                    XDrawLine( dpy, t->title_w, Scr.Gray1GC, 0, lp, t->frame_w, lp );
 			}
 			else
 				XDrawLine( dpy, t->title_w, Scr.BlackGC,
 						  4, lp-1, t->frame_w-7, lp-1 );
-		}
-		if( Scr.d_depth>1 ){
```
Comment (morgant) [inline mlvwm/borders.c:504]: I'm having trouble visualizing what part this drew, but I can't tell the difference without it so should be fine to remove.

File: mlvwm/mlvwm.h [hunk 2]
```diff
@@ -178,9 +178,9 @@ extern Atom _XA_WM_DESKTOP;
 
 extern void Done( int, char * );
 #ifdef USE_LOCALE
-#define DEFAULTFS "-adobe-*-*-r-*-*-14-*-*-*-p-*-*-*,\
+#define DEFAULTFS "-adobe-*-*-r-*-*-12-*-*-*-p-*-*-*,\
 -*-*-*-r-*-*-14-*-*-*-*-*-*-*"
```
Comment (morgant) [inline mlvwm/mlvwm.h:182]: Looks like you missed changing this line to 12. It's just a fail-over if it can't find Adobe fonts, but might come up.

**Comment by morgant** [comment]
Looks great with the smaller font size and the changed stripes for default System7 theme.

I see what you mean about the title bar being a different height. It's fine for now and better than it was, but I'm leaning towards the title bar height (`TITLE_HEIGHT`) being set to the smaller size and then just expanding it for System8 theme, like is done for window size in `AddWindow()`. We could tackle this separately, if you'd like.

Along with the menu bar height being reduced to 20px, I think the scroll bars should be reduced to 16px. I don't know if that's something you want to try to implement in this PR or you'd rather I worked on some of that sort of stuff after merging this.

File: mlvwm/borders.c [hunk 3]
```diff
@@ -488,30 +488,16 @@ void SetTitleBar( MlvwmWindow *t, Bool on_off )
...
-			if( Scr.flags&SYSTEM8 )
-				DrawShadowBox( 4, 14, t->frame_w-15, 2, t->title_w, 1,
-							  Scr.WhiteGC, Scr.Gray1GC, SHADOW_ALL );
-			else
-				XDrawLine( dpy, t->title_w, Scr.Gray1GC,
-						  4, 15, t->frame_w-9, 15 );
-		}
-		else
```
Comment (morgant) [inline mlvwm/borders.c:512]: Did you mean to strip out the lines on monochrome displays?

Comment (morgant) [reply mlvwm/borders.c:512]: I've restored this one bit of code, updated to match the lines drawn on colour displays (see commit e0823a6a150e830cd119eef1c8d6140f77efb0aa).

**Comment by morgant** [comment]
As noted in a code comment, I've restored a couple lines which were stripped out which rendered the stripes on monochrome displays (adjusted to match the System7 styling as you had updated it; see commit e0823a6a150e830cd119eef1c8d6140f77efb0aa) and merged this.

Huge thanks for hacking on MLVWM and improving the appearance, @TheMorc!

**Pull request merged**